### PR TITLE
message retreive in chat, link to gitter login

### PR DESCRIPTION
**Pull request by bartist**

### CODE REVIEW

**Comment by bartist** [comment]
@steffansluis @WouterKvG please check

**Comment by steffansluis** [comment]
@bartist The code looks good, LGTM!

File: addon/templates/components/chat-window.hbs [hunk 1]
```diff
@@ -2,36 +2,44 @@
 <div class="{{windowSize}}">
   <div class="chat-top-bar">
     {{#unless openChat}}
-    <div class="chat-small"  {{action 'toggle'}}>
-      Open Chat
-    </div>
+      <div class="chat-small"  {{action 'toggle'}}>
```
Comment (WouterKvG) [inline addon/templates/components/chat-window.hbs:5]: Are these spaces necessary?

**Comment by WouterKvG** [comment]
Apart from the added spaces I think this is ready to merge

File: addon/templates/components/chat-window.hbs (hunk 1 above)
Comment (bartist) [inline addon/templates/components/chat-window.hbs:5]: necessary: no
handy for overview: absolutely

**Pull request merged**